DEVELOPING_OPAL/validate/src/test/java/org/opalj/fpcf/fixtures/string/ArrayOps.java

```diff
@@ -0,0 +1,56 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj.fpcf.fixtures.string;
+
+import org.opalj.fpcf.fixtures.string.tools.StringProvider;
+import org.opalj.fpcf.properties.string_analysis.*;
+
+/**
+ * @see SimpleStringOps
```

Comment:
Should give at least a short explanation of what test cases to expect in the file, for each of these test files.